templates/single-meetings.php

```diff
@@ -20,17 +14,9 @@ function inactive(x) {
 	$meeting_name = $meeting->group;
 }
 
-/*
-if (!empty($meeting->end_time)) {
-	$myEndTime = $meeting["end_time"];
-} else {
-	$myEndTime = add_hour_to_start_time(@$meeting->time);
-}
-*/
-
 //define some vars for the map
 wp_localize_script('tsml_public', 'tsml_map', [
-	'directions' => __('Directions', '12-step-meeting-list-feedback-enhancement'),
+	'directions' => __('Directions', '12-step-meeting-list'),
```

Comment:
Does this affect, or have the potential to affect, TSML?

Reply:
it's ok but i'm not sure the rationale for changing it?

Reply:
i'm not sure this will work. from https://pascalbirchler.com/text-domain-wordpress-internationalization/ 

> In the WP-CLI command we automatically try to guess your plugin’s slug (and thus the text domain) from the folder name. After that, it only extracts gettext calls with that text domain. Any other text domain will be ignored. This means it finds and extracts __( 'Translate me', 'my-plugin' ), but skips __( 'Translate me', 'another-plugin' ).

Reply:
Thanks for pointing that out @joshreisner. It worked correctly in English. My rationale was to make syncing with TSML's single-meeting template file easier as the Feedback Enhancement code is pretty much appended after the TSML code. I'll revert the code back to the correct plugin.